plugins/fortinet_fortigate/help.md

```diff
@@ -719,13 +719,13 @@ Add the orchestrator's IP address to the trusted hosts in CIDR form e.g. `198.51
 
 # Version History
 
-* 6.0.1 - Resolve connection test failure with Fortigate version 6.4.1 and above
+* 6.0.1 - Resolve connection test failure with Fortigate version 6.4.1 and above | Fix bug in action Check if Address in Group where if the IPV6 Group did not exist, the action would fail
```

Comment:
Reword -> Fix bug in action 'Check if Address in Group', where the action would fail if the IPV6 Group did not exist